packages/modals/src/elements/Header.tsx

```diff
@@ -6,8 +6,17 @@
  */
 
 import React from 'react';
-import { StyledHeader, IStyledHeaderProps } from '../styled';
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import AlertErrorStrokeIcon from '@zendeskgarden/svg-icons/src/16/alert-error-stroke.svg';
 import { useModalContext } from '../utils/useModalContext';
+import { StyledHeader, IStyledHeaderProps } from '../styled';
+
+const StyledIcon = styled(AlertErrorStrokeIcon)`
```

Comment:
Should this styled element style the StrokeIcon directly or style a div which positions its contents (the icon) correctly? 

I'm leaning towards the styled div solution as it lets you move this files to the `styled` directory and use multiple icon variants if it's something we want in the future.

Reply:
Awesome, this component has been moved to the `/styled` directory. The styling is still applied to the StrokeIcon directly. This component can be adjusted accordingly when the requirements change.